src/_layouts/release.md

```diff
@@ -1,23 +1,20 @@
 ---
 layout: default_md
-title: ActiveMQ 5.15.13 Release 
-title-class: page-title-activemq5
-type: activemq5
 ---
 
-[Overview](overview) > [Download](download) > [ActiveMQ 5.15.13 Release](activemq-51513-release)
+[Overview](overview) > [Download](download) > [ActiveMQ {{page.version}} Release](activemq-51513-release)
 
-ActiveMQ 5.15.13 Release
+ActiveMQ {{page.version}} Release
 -----------------------
 
-Apache ActiveMQ 5.15.13 includes several resolved[ issues](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12311210&version=12347002) and bug fixes.
+{{content}}
 
 ### Getting the Binary Distributions
 
 Description|Download Link|_Verify_
 ---|---|---
-Windows Distribution|[apache-activemq-5.15.13-bin.zip](https://archive.apache.org/dist/activemq/5.15.13/apache-activemq-5.15.13-bin.zip)|[ASC](https://archive.apache.org/dist/activemq/5.15.13/apache-activemq-5.15.13-bin.zip.asc), [SHA512](https://archive.apache.org/dist/activemq/5.15.13/apache-activemq-5.15.13-bin.zip.sha512)
-Unix/Linux/Cygwin Distribution|[apache-activemq-5.15.13-bin.tar.gz](https://archive.apache.org/dist/activemq/5.15.13/apache-activemq-5.15.13-bin.tar.gz)|[ASC](https://archive.apache.org/dist/activemq/5.15.13/apache-activemq-5.15.13-bin.tar.gz.asc), [SHA512](https://archive.apache.org/dist/activemq/5.15.13/apache-activemq-5.15.13-bin.tar.gz.sha512)
+Windows Distribution|[apache-activemq-{{page.version}}-bin.zip](https://archive.apache.org/dist/activemq/{{page.version}}/apache-activemq-{{page.version}}-bin.zip)|[ASC](https://archive.apache.org/dist/activemq/{{page.version}}/apache-activemq-{{page.version}}-bin.zip.asc), [SHA512](https://archive.apache.org/dist/activemq/{{page.version}}/apache-activemq-{{page.version}}-bin.zip.sha512)
+Unix/Linux/Cygwin Distribution|[apache-activemq-{{page.version}}-bin.tar.gz](https://archive.apache.org/dist/activemq/{{page.version}}/apache-activemq-{{page.version}}-bin.tar.gz)|[ASC](https://archive.apache.org/dist/activemq/{{page.version}}/apache-activemq-{{page.version}}-bin.tar.gz.asc), [SHA512](https://archive.apache.org/dist/activemq/{{page.version}}/apache-activemq-{{page.version}}-bin.tar.gz.sha512)
```

Comment:
The archive links should go the download mirrors for the current release(s), not the archive, like the main download page. See the existing 5.16.1 release page this change would replace, for example. Same for the source links below.

Reply:
Correct: for "active" download (basically, the one located on https://dist.apache.org/repos/dist/release/activemq/), the download link is not archive.
Archive is used for "old" release (automatically copied from dist): https://archive.apache.org/dist/activemq/